src/adapters/appnexus.js

```diff
@@ -118,7 +118,11 @@ AppNexusAdapter = function AppNexusAdapter() {
 
     //append referrer
     if (referrer === '') {
-      referrer = utils.getTopWindowUrl();
+      try {
+        referrer = utils.getTopWindowUrl();
```

Comment:
I'm not sure everyone will realize this can throw. Is there a way to swallow the error in `utils.getTopeWindowUrl` ?